docs/tutorials/01_introduction_cloud_runtime.ipynb

```diff
@@ -228,44 +228,48 @@
       "  Name: sampler\n",
       "  Description: Generates quasi-probabilities by sampling quantum circuits.\n",
       "  Creation date: 2021-10-26T14:41:57Z\n",
-      "  Update date: 2022-02-24T22:17:10.144954Z\n",
+      "  Update date: 2022-03-14T17:12:06.298047Z\n",
       "  Max execution time: 10000\n",
       "  Backend requirements:\n",
       "    none\n",
       "  Input parameters:\n",
       "    Properties:\n",
+      "        - circuit_indices:\n",
+      "            Description: Indexes of the circuits to evaluate.\n",
```

Comment:
Pretty minor but I'd suggest being consistent with the plural of "index"

Reply:
I agree, likely nothing to fix in this PR because that's just output returned from the server.

Reply:
Fixed the tutorial as well as opened a PR upstream to fix this.